src/Routes.js

```diff
@@ -85,33 +86,51 @@ const paths = [
   { title: 'Topics', path: '/topics', component: TopicsList },
   { title: 'Topic details', path: '/topics/:id', component: TopicDetails },
 ];
+const INVENTORY_TOTAL_FETCH_URL = '/api/inventory/v1/hosts';
 
-export const Routes = () => (
-  <Suspense
-    fallback={
-      <Bullseye>
-        <Spinner size="xl" />
-      </Bullseye>
+export const Routes = () => {
+  const [hasSystems, setHasSystems] = useState(true);
```

Comment:
Sorry to comment on a draft, but there should somewhere in advisor already be a zero/empty state. There is an account called `empty-account` (pw see slack), that should show you the page that should be adjusted.

Reply:
I know we addressed it on stand up, but advisor is a special case that currently has no zerostate, only an empty state. Thanks for the info that, I was borrowing someone else's account